CONTRIBUTING.md

```diff
@@ -57,10 +78,33 @@ pnpm ci:test
 
 This will run `services:run`, `test` and then `services:clean`
 
-> The tests may break if you are running your tests locally using`services:run` in a separate terminal.
+> The tests may break if you are running your tests locally using `services:run` in a separate terminal.
 > To fix this run `services:clean` to clean docker containers and volumes.
 
-## Commit Convention
+# Using local packages
+
+Simply `add` the package folder on your desired project:
+
+```sh
+cd my-app
+pnpm add ~/path/to/my/fuel-ts/packages/fuels
+```
+
+Or, if you want to try something globally:
+
+```sh
+pnpm add --global ~/path/to/my/fuel-ts/packages/fuels
+```
+
+## Important
+
+When using your local version of `fuels`, remember that you need to run `pnpm build` on every change you do to source files, otherwise the chagnes won't be propagated to dependent apps using it.
```

Comment:
```suggestion
When using your local version of `fuels`, remember that you need to run `pnpm build` on every change you do to source files, otherwise the changes won't be propagated to dependent apps using it.
```